docs/site/Where-filter.md

```diff
@@ -8,4 +8,560 @@ file: packages/metadata/README.md
 sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Where-filter.html
 summary: A <i>where</i> filter specifies a set of logical conditions to match, similar to a WHERE clause in a SQL query.
----
\ No newline at end of file
+---
+
+## REST API
+
+In the first form below, the condition is equivalence, that is, it tests whether _property_ equals _value_.
+The second form below is for all other conditions.
+
+```
+filter[where][property]=value
+```
+
+```
+filter[where][property][op]=value
+```
+
+For example, if there is a cars model with an `odo` property, the following query finds instances where the `odo` is greater than 5000:
+
+```
+/cars?filter[where][odo][gt]=5000
+```
+
+For example, here is a query to find cars with `odo` is less than 30,000:
+
+```
+/cars?filter[where][odo][lt]=30000
```

Comment:
i know it's not part of the LB3 docs either. It might be helpful to "join" the two conditions. e.g. what should be the `where` clause if the value ranges from 10000 and 30000. It's out of scope of this PR though.

Reply:
Good idea, I think Rifa mentioned it before. I think `Querying-data.md` will be a good place to introduce the function.